data/com.github.jeromerobert.pdfarranger.desktop

```diff
@@ -1,30 +1,31 @@
 [Desktop Entry]
 Version=1.0
 Name=PDF Arranger
+Name[ca]=Arranjador de PDF
 Name[hr]=PDF uređivač
```

Comment:
You did not translate the name. That's fine with me if it happened intentionally.

Reply:
Yes, I did it intentionally, and I put in alphabetical order languages.